### DISCUSSION TITLE
FPS Manager and other new options

### DISCUSSION THREAD

**Discussion by Tirlititi** [Albeoris/Memoria#209] [General]
In ``Memoria.ini``:
- The framerate at which the game runs can be customised, in order to have a smoother game and menus; shout-out to [Iamgoofball](https://github.com/Albeoris/Memoria/issues?q=is%3Apr+is%3Aopen+author%3AIamgoofball) for preparing the groundwork for it
- The standard speed of the game ("Ticks per second") can be changed per module (field / battle / world map)
- A new section ``[SaveFile]`` gives options to control auto-saves and automatic Cloud saves
- A new option ``[Battle] LockEquippedAbilities`` changes the abilities available to be only those on currently equipped gears
- A new option ``[Battle] PersistentDangerValue`` prevents the danger value (for random encounters) to reset in screen transitions
- The option ``[Export] Battle`` now generates the enemy attack sequences as ``.seq`` files; see [there](https://github.com/Albeoris/Memoria/wiki/Battle-SFX-Sequence)

Other modding options:
- The Voice Acting system got improved and allows to play voice sounds for battle dialogs; shout-out to [barkermn01](https://github.com/Albeoris/Memoria/commits?author=barkermn01) as always; see [there](https://github.com/Albeoris/Memoria/wiki/Voice-acting-system)
- A ``BattlePatch.txt`` file can be included in mod folders in order to modify battles on top of the binary database and with extra features; see [there](https://github.com/Albeoris/Memoria/wiki/Battle-Patch)
- The mod manager improved a bit
- A part of the world map database has been externalised to the folder ``StreamingAssets/Data/World/``: these files allow to define custom ambiant colors, custom vehicle specifications and custom conditions for the world map changes and effects
- The options ``WorldMusicList`` and ``MoogleFieldList`` are added to ``DictionaryPatch.txt``; see [there](https://github.com/Albeoris/Memoria/wiki/Dictionary-Patch)
- The option ``[Memoria] LauncherLanguage`` in ``Settings.ini`` allows to pick the language for the launcher window independantly of the system's language

Also:
- The application's name displays the current map informations more accuratly (mainly visible in windowed mode)
- The damage figures are always displayed in the frame of the screen, similar to the PSX version's behaviour
- Various bug fixes

**Tips:** Before running the patcher, make sure to (1) make a backup of your ``Memoria.ini`` file, as the new version may replace the older one and erase your custom settings, and (2) first disable the option ``Fix Steam Overlay`` in the launcher in order to update the launcher application as well (and then you can re-enable it afterwards)

<hr /><em>This discussion was created from the release <a href='https://github.com/Albeoris/Memoria/releases/tag/'>FPS Manager and other new options</a>.</em>

**Comment by livnitro** [comment]
what would be the correct way to set my launcher's language to english, if i may ask?

**Reply by Albeoris** [reply]
@livnitro
* You can change game language in the game (on the title screen)
* You can change launcher language in the `Settings.ini`
```ini
[Memoria]
LauncherLanguage = en
```
